capstone/timeline.md

```diff
@@ -5,43 +5,45 @@ subtitle: For MDS Capstone Projects
+| [Students rate the projects](#students-rate-the-projects)                                      | End-January, 2023     |
+| [Partners demonstrate readiness of data and sign legal docs](#partners-demonstrate-readiness-of-data-and-sign-legal-docs)                                    | February 1, 2023      |
+| Students are assigned to projects                                                              | Feb - March, 2023     |
+| [Kickoff meetings](#kickoff-meetings)                                                          | April, 2023           |
+| [Project](#project)                                                                            | May 1 - June 29, 2023 |
+| [Kickoff hackathon](#hackathon)                                                                | May 1 - May 3, 2023   |
 
 <br>
 
 ## Proposal revision process
 
 Members of an organization interested in participating as a capstone partner should submit a project [proposal](https://ubc-mds.github.io/capstone/proposal/).
 
-Proposals received *before November 15th* will be reviewed by the MDS teams and feedback/suggestions for revisions will be provided to the partner, who can then revise and re-submit their proposal. This revision process aims to maximize the chance of a proposal being selected by students. Proposals received *after November 15th* will not be provided with feedback.
+Proposals received *before November 1st* will be reviewed by the MDS teams and feedback/suggestions for revisions will be provided to the partner, who can then revise and re-submit their proposal. This revision process aims to maximize the chance of a proposal being selected by students. Proposals received *after November 1st* will not be provided with feedback.
 
 ## Selection of proposals
 
 The MDS leadership team will select proposals based primarily on student interest, and will invite the successful organizations to be a capstone partner.
 
-## Partners accept invitations
-
-The successful capstone partners accept the offers, and confirm their attendance at the capstone fair.
+## Partners submit video presentations
+Capstone partners should submit video presentations (5-10 minutes) to provide information additional to the proposals. This will help students review your project(s) before attending the capstone fair.
 
 ## Capstone fair
 
-Capstone partners must attend the Capstone Fair to meet with the students and answer any questions they may have. The Capstone Fair takes place at the UBC Robson Square campus in downtown Vancouver. Detailed information will be emailed to each capstone partner. An example of a Capstone fair slide deck can be found [here](/capstone/Sauder2019CapstoneFair.pdf).
+Capstone partners must attend the Capstone Fair to meet with the students and answer any questions they may have. The Capstone Fair takes place at the UBC Robson Square campus in downtown Vancouver. Detailed information will be emailed to each capstone partner. 
```

Comment:
I heard from Milad that there will be a Zoom session too for non-local partners. How about explicitly putting it here? Otherwise it will highly discourage remote partners to submit their proposals.

Reply:
I'll double check with Milad and update this.